bash-preexec.sh

```diff
@@ -143,15 +150,22 @@ __bp_precmd_invoke_cmd() {
       return
     fi
     local __bp_inside_precmd=1
+    bash_preexec_invoke_precmd_functions "$__bp_last_ret_value" "$__bp_last_argument_prev_command"
+}
 
+# This function invokes every function defined in our function array
+# "precmd_function".  This function receives the arguments $1 and $2 for $? and
+# $_, respectively, that will be set for the precmd functions.
+bash_preexec_invoke_precmd_functions() {
+    local __lastexit=$1 __lastarg=$2
```

Comment:
Don't think the underscore naming scheme is necessary here.

Reply:
dda9193